test/integration/fastify/fastify.tap.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+
+const helper = require('../../lib/agent_helper')
+
+tap.test('Fastify Instrumentation', (t) => {
+  t.autoend()
+
+  let agent = null
+  let fastifyExport = null
+
+  t.beforeEach(() => {
+    agent = helper.instrumentMockedAgent({
+      feature_flag: {
+        fastify_instrumentation: true
+      }
+    })
+    fastifyExport = require('fastify')
```

Comment:
don't we want to run 3 diff tests here?
```
fastifyExport = require('fastify')
{ fastify: namedExport } = require('fastify')
{ default: defaultExport } = require('fastify')
```

Reply:
The test included is simply asserting the properties are propogated as-is from the original. If you look at the assertions within.

While I could do the above, it would be tied specifically to v3 and inflexible and not particularly providing more value.

Reply:
Ok, makes sense.  My only concern is we don't have explicit coverage for the named and default exports, even if it's only on version 3

Reply:
We certainly can add "explicit" coverage but it doesn't need to be via additional requires. As it currently is, you have *more* coverage than just those two properties. The intent was to get a quick fix out to help the customer until we we gave this a deep-dive with the MMF.

I'm fine adding explicit tests, though. What I would do in that case is rip this file out of integraion tests, move into versioned only and then add an expilcit v3 test runner.